models/engine/db_storage.py

```diff
@@ -74,3 +74,43 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Get a single object from the database
+
+        Args:
+            cls     - string representing the class name
+            id      - string representing the object ID
+
+        Returns:
+            Object base on the class and id or else None.
+        """
+        if not cls or not id:
+            return None
+        if type(cls) is str and cls not in classes.keys():
+            return None
+        elif type(cls) is not str and cls not in classes.values():
+            return None
+
+        if type(cls) is str:
+            cls = classes[cls]
+        return self.__session.query(cls).filter(cls.id == id).first()
+
+    def count(self, cls=None):
+        """returns the count of all objects or specific class in database
+
+        Arsg:
+            cls - Default (None) else String representing the class name
+
+        Returns:
+            `count` of all object in __objects is cls is None, else `count`
+            of the specific onbject in __object.
+        """
+        if not cls:
+            count = 0
```

Comment:
Could this be Pythonically condensed using list comprehension? `return sum(len(self.__session.query(obj).all()) for obj in classes.values())`